buildutils/torque.spec.in

```diff
@@ -61,9 +61,9 @@
 %define ac_trqauthd_sock_dir %{?trqauthd_sock_dir:%{trqauthd_sock_dir}}%{!?trqauthd_sock_dir:/tmp}
 
 ### Build Requirements
-%define breq_cpuset   %{?with_cpuset:hwloc-devel}
+%define breq_cpuset   %{!?hwloc_path:%{?with_cpuset:hwloc-devel}}
 %define breq_gui      %{?with_gui:tcl-devel tk-devel tclx}
-%define breq_numa     %{?with_cpuset:hwloc-devel}
+%define breq_numa     %{!?hwloc_path:%{?with_cpuset:hwloc-devel}}
 %define breq_munge    %{?with_munge:munge-devel}
```

Comment:
Yes and no. I understand your point, but I think you did not (entirely) answer one of my questions; AFAICT, in my scenario here, hwloc-devel will never be installed on another system than my build environment (and even not there, since I compile from source). So the torque rpm in this case effectively seems to have no dependency at all on hwloc-devel. At build time it is otherwise provided, at install time, it is no longer relevant.

Building a separate rpm for hwloc-devel is probably a good idea (as I said earlier), but not really necessary either _I THINK_ (I may be very wrong here once I actually test the cpuset/cgroup features on our el6 cluster). For package infrastructures, the case is different, but they will (should?) never have a `--with-hwloc-path` ...